packages/fields/src/fields/featured-image/featured-image-edit.tsx

```diff
@@ -3,9 +3,9 @@
  */
 import { Button, __experimentalGrid as Grid } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { useCallback, useRef } from '@wordpress/element';
+import { useCallback, useRef, useState } from '@wordpress/element';
 // @ts-ignore
```

Comment:
Should this be `/* tslint:disable-next-line */`? Sorry, that's all I've got tonight 😄